kit/inputs/src/UIKeyTextView.tsx

```diff
@@ -47,6 +53,10 @@ export function useKeyTextView(
             if (hasInvalidChars !== isCharsInvalid) {
                 setHasInvalidChars(isCharsInvalid);
             }
+
+            if (onChangeKey) {
```

Comment:
Same as above, using `onChangeText` you may not write this additional code

Reply:
https://github.com/tonlabs/UIKit/pull/1476/commits/9a0447c5cc35417b0ecb3622a1aaebf571c8e06b